src/main/java/io/asyncer/r2dbc/mysql/MySqlConnectionConfiguration.java

```diff
@@ -245,29 +256,31 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
-        return Objects.hash(isHost, domain, port, ssl, tcpKeepAlive, tcpNoDelay,
-                            connectTimeout, socketTimeout, serverZoneId, zeroDateOption, user, password, database,
-                            preferPrepareStatement, queryCacheSize, prepareCacheSize, extensions, passwordPublisher);
+        return Objects.hash(isHost, domain, port, ssl, tcpKeepAlive, tcpNoDelay, connectTimeout,
+            socketTimeout, serverZoneId, zeroDateOption, user, password, database, createDatabaseIfNotExist,
+            preferPrepareStatement, queryCacheSize, prepareCacheSize, extensions, passwordPublisher);
     }
 
     @Override
     public String toString() {
         if (isHost) {
             return "MySqlConnectionConfiguration{, host='" + domain + "', port=" + port + ", ssl=" + ssl +
-                   ", tcpNoDelay=" + tcpNoDelay + ", tcpKeepAlive=" + tcpKeepAlive + ", connectTimeout=" +
-                   connectTimeout + ", socketTimeout=" + socketTimeout + ", serverZoneId=" + serverZoneId +
-                   ", zeroDateOption=" + zeroDateOption + ", user='" + user + '\'' + ", password=" + password +
-                   ", database='" + database + "', preferPrepareStatement=" + preferPrepareStatement +
-                   ", queryCacheSize=" + queryCacheSize + ", prepareCacheSize=" + prepareCacheSize +
-                   ", extensions=" + extensions + ", passwordPublisher=" + passwordPublisher + '}';
+                ", tcpNoDelay=" + tcpNoDelay + ", tcpKeepAlive=" + tcpKeepAlive + ", connectTimeout=" +
+                connectTimeout + ", socketTimeout=" + socketTimeout + ", serverZoneId=" + serverZoneId +
+                ", zeroDateOption=" + zeroDateOption + ", user='" + user + '\'' + ", password=" + password +
+                ", database='" + database + ", createDatabaseIfNotExist=" + createDatabaseIfNotExist +
+                "', preferPrepareStatement=" + preferPrepareStatement + ", queryCacheSize=" + queryCacheSize +
```

Comment:
seems closing ' is misplaced.
```suggestion
                ", preferPrepareStatement=" + preferPrepareStatement + ", queryCacheSize=" + queryCacheSize +
```